.github/workflows/build-action.yml

```diff
@@ -65,7 +65,7 @@ jobs:
        - name: Environment variables
          run: sudo -E bash -c set
        - name: Install required packages
-         run: sudo apt-get install git curl unzip libarchive-zip-perl dos2unix xz-utils kpartx
+         run: sudo apt-get install git curl wget unzip libarchive-zip-perl dos2unix xz-utils kpartx systemd-container zerofree qemu-user-static
```

Comment:
are these really needed, the qemu one in particular ?

Reply:
Yes, qemu is what allows it to run the machine without being on an arm platform.

Reply:
Yes please keep qemu in. Qemu tooling is actually used to mount the image when build.bash as a non-root user. @mstormi there is no emulation at all, just using the qemu helpers to do the userspace mount.